src/ansys/fluent/core/codegen/datamodelgen.py

```diff
@@ -66,6 +66,34 @@
 )
 
 
+def _write_meshing_utilities_stub(file_path):
+    if file_path.exists():
+        file_path.unlink()
+    file = open(file_path, "w", encoding="utf8")
+    file.write("#\n")
+    file.write("# This is an auto-generated file.  DO NOT EDIT!\n")
+    file.write("#\n")
+    file.write("# pylint: disable=line-too-long\n\n")
+    file.write("from ansys.fluent.core.services.datamodel_se import PyMenu\n")
+    file.write("from typing import *\n")
```

Comment:
We can now change `List` -> `list` and `Dict` to `dict` in `_PY_TYPE_BY_DM_TYPE` and remove the above import.

Reply:
Done. Thank you.